pkg/shell/cockpit-networking.js

```diff
@@ -2968,22 +2970,23 @@ PageNetworkBridgePortSettings.prototype = {
             options.hairpin_mode = hairpin_mode_input.prop('checked');
         }
 
+        /* TODO: Migrate away from jQuery spaghetti code */
```

Comment:
possibly add comment about adding linking code between labels and input controls when this is cleaned up